Cargo.toml

```diff
@@ -14,3 +14,15 @@ members = [
     "frontend_server"
 ]
 default-members = ["aster"]
+
+
+[workspace.dependencies]  
+tokio = "1.28.2"
+anyhow = "1.0.71"
+async-trait = "0.1.68"
+chrono = "0.4.26"
+log = "0.4.18"
+crossbeam-channel = "0.5.8"
```

Comment:
We should be wary of which of #100 and this PR merges first, since this dep changes.